core/src/main/resources/hudson/triggers/TimerTrigger/help-spec.html

```diff
@@ -78,4 +78,31 @@
 # once a day on the 1st and 15th of every month except December
 H H 1,15 1-11 *
 </pre>
+<p>
+  <b>Timezone specification</b>. <br/>
+  Periodic tasks are normally executed at the scheduled time in the timezone of 
+  the Jenkins master JVM (currently Central European Time). 
+  This behaviour can be <b>optionally</b> changed by specifying an alternative timezone in the
+  first line of the field.
+</p>
+<ul>
+  <li>Timezone specification should contain the &quot;TZ=$(timezoneID)&quot; string</li>
```

Comment:
*should* looks like a soft recommendation, when it fact it's a hard requirement. Otherwise it's not a TZ specification. Also, it does not *contain* this string, this is what the entire thing looks like.